src/components/subcomponents/MainDiv.js

```diff
@@ -0,0 +1,9 @@
+function MainIDivjs() {
```

Comment:
You've named the file `MainDiv`, and you've imported it as `MainDiv` where it's being used, but this component is called `MainIDivjs` - I recommend renaming this function to `MainDiv` (and the export default at the bottom) to keep things consistent.

On that note, the name `MainDiv` isn't very descriptive. I'd probably go with something more like `Buttons`